spec/turn_spec.rb

```diff
@@ -0,0 +1,202 @@
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+
+RSpec.describe Player do
```

Comment:
This `describe` should represent the class that we are testing. In this spec, we are testing the `Turn` class, not the `Player` class.

```ruby
RSpec.describe Turn do
```